blockchain_exploration/exploration.py

```diff
@@ -152,6 +161,8 @@ def get_explorer_url_for_token(network: str,
     elif network == Network.SUI:
         # token_id => object_id
         return f"{base_path}/object/{token_id}"
+    elif network == Network.TON:
+        return f"{base_path}/not-implemented/"
```

Comment:
Given this is not implemented, maybe base to not add it so that we can fix it rather than silently ignore...

Or, maybe just return `None` so that we can better handle it in the calling code?